app/services/cart_items/index.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module CartItems
+  class Index
+    def initialize(current_user)
+      @current_user = current_user
+    end
+
+    def items_in_cart
+      CartItem.all.map do |cart_item|
```

Comment:
em.... WHAT? You wanna use `map` for whole table? what happens if there will be 1B records?